server/modules/reorder.py

```diff
@@ -0,0 +1,36 @@
+from .moduleimpl import ModuleImpl
+import json
+
+
+class ReorderFromTable(ModuleImpl):
+    @staticmethod
+    def render(wf_module, table):
+        # Error message for when reorder history is corrupted
+        CORRUPT_HISTORY_ERROR = 'Reorder history is corrupted. Did you remove one of the input columns?'
```

Comment:
Does it not handle removal of input columns? It needs to, this will happen all the time.